runtime.py

```diff
@@ -6,11 +6,17 @@
 CLIENT_REALM = os.getenv("CLIENT_REALM")
 RUN_ID = os.getenv("RUN_ID")
 
-inputs_list = [CLIENT_ID, CLIENT_KEY, CLIENT_REALM, RUN_ID]
+inputs_list = [CLIENT_ID, CLIENT_KEY, CLIENT_REALM]
 
 if None in inputs_list:
-    print("- Some mandatory input is empty. Please, check the input list.")
+    print("- Some mandatory authentication input is empty. Please, check the input list.")
     exit(1)
+    
+if RUN_ID == "":
+    print("- RUN_ID was not provided.")
+    print("  Deployment was not successfully created.")
```

Comment:
We don't know the reason why the RUN_ID might be empty, so this message might not be true depending on the context of the client pipeline.